css-ecommerce-page/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header">
+    <header>
+      <p>New Arrivals</p>
+    </header>
+  </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-2">
+        <img src="images/dress1.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/ysl-bag.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/key.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/lv-bag.jpg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/dress2.jpeg" alt="dress1">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-2">
+        <p>Free people <span> |Size 6</span></p>
+      </div>
+      <div class="col-2">
+        <p>Yves Saint Laurent <span> |One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Tiffany <span> |One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Louis Vuitton <span>|One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Reformation <span> |Size 4</span></p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-2">
+        <p><span class="peach-background color-black">$35</span>&nbsp;<span><s>$125 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$200</span>&nbsp;<span><s>$1600 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$476</span>&nbsp;<span><s>$2000 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$95</span>&nbsp;<span><s>$1200 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$20</span>&nbsp;<span><s>$198 retail</s></span></p>
+      </div>
+    </div>
```

Comment:
Content-wise, related information should be grouped together as much as possible. For example, a product's image, name, size, and pricing are all related, so their grouping should take priority over, for example, grouping all the product images together.

---

![image](https://user-images.githubusercontent.com/16143771/152214469-7189dd2f-0bc4-44f0-a386-5ebc92c37086.png)